frame/nfts/src/types.rs

```diff
@@ -182,39 +198,101 @@ pub enum CollectionSetting {
 	UnlockedMetadata,
 	/// Attributes of this collection can be modified.
 	UnlockedAttributes,
+	/// The supply of this collection can be modified.
+	UnlockedMaxSupply,
 	/// When this isn't set then the deposit is required to hold the items of this collection.
```

Comment:
Any chance we can rephrase this line (even though it's not changed)? When we don't set DepositRequired then a deposit is required to hold items of this collection? Maybe there's a way to phase this in terms of sufficiency? At the moment it's not that clear.

Reply:
I would propose changing it to `MutableMaxSupply`, but not so sure the regular users would understand that